src/quicklook.cs

```diff
@@ -176,8 +181,11 @@ interface QLPreviewItem {
 #endif
 	}
 
+	[NoMac]
 	delegate bool QLPreviewReplyDrawingHandler (CGContext context, QLPreviewReply reply, out NSError error);
+	[NoMac]
```

Comment:
These delegates are also in the quicklookui.cs file - you can delete those in quicklookui.cs, and remove the [NoMac] attribute here

Reply:
Removed!